js/content/community.js

```diff
@@ -291,6 +292,260 @@ let CommentHandler = (function(){
         observer.observe(document.body, { childList: true });
     };
 
+    function loadScripts() {
+        return new Promise(async function(res) {
+            scriptsLoaded = true;
+            await ExtensionLayer.loadInPageContext("//steamcommunity-a.akamaihd.net/public/shared/javascript/shared_global.js?v=NuCEF6NW8c0Q");
```

Comment:
Why are you waiting for the previous script's `onload` event? `<script>`s added to the page will be executed in the order they appear in the DOM, so if you add all of these they will load in order.

What is the `?v=` parameter for? It looks like a session key or validation token that is expected to change on a schedule. (Mine are different already.)

My akamai requests also embed the language in the querystring and we should do the same with these.

Reply:
I have to wait for the code to be fully executed before adding the next script. The version parameter is not personalized, but does help to avoid Valve (breaking) code changes. I avoided the language parameter, because I don't want it to be english only (e.g. for error messages).